boundary_condition_vector.m

```diff
@@ -144,7 +144,8 @@
                 fListL = bcL{d1};
                 fListR = bcR{d1};
                 
-                bcVec = bcVec + combine_dimensions_D(pde.deg,fListL,timeFacL,hash_table,opts.use_oldhash);
+                partial = combine_dimensions_D(pde.deg,fListL,timeFacL,hash_table,opts.use_oldhash);
```

Comment:
What's this `partial` variable been introduced for?

I'm unsure why we are modifying this file at all for a gold data PR?

Reply:
this was simply so I could examine the result of the combine_dimensions_D call as I was developing